src/dev/build/tasks/os_packages/docker_generator/templates/ironbank/hardening_manifest.yaml

```diff
@@ -48,9 +48,9 @@ resources:
 
 # List of project maintainers
 maintainers:
-  - email: 'tyler.smalley@elastic.co'
-    name: 'Tyler Smalley'
-    username: 'tylersmalley'
+  - email: 'jon@elastic.co'
+    name: 'Jonathan Budzenski'
+    username: 'jbudz'
     cht_member: false
   - email: 'klepal_alexander@bah.com'
```

Comment:
What about these users? The original ticket mentions them:
>For example, neither Alexander Klepal nor Yalabe Dukuly are not on the Kibana team.

However, they've never been on Kibana apparently, they have external e-mails.